docs/VictoriaLogs/FAQ.md

```diff
@@ -175,3 +175,21 @@ And for the following log, its `_msg` will be `foo bar in body`:
   "body": "foo bar in body"
 }
 ```
+
+## What is the max size of a log record?
+
+VictoriaLogs is optimized for logs with records of rather small length.
```

Comment:
Let's remove this sentence, since it may mislead readers in the sense that VictoriaLogs doesn't work with moderately sized log records (e.g. a few KB or tens KB length). VictoriaLogs fits well the majority of  cases for logs. Log records with the length of a MB and more is very specific and rare case.